recipes-core/rauc/rauc/mx8mm/system_emmc.conf

```diff
@@ -1,47 +1,48 @@
 [system]
 compatible=@MACHINE@
-bootloader=uboot
+bootloader=@BOOTLOADER@
 mountprefix=/mnt/rauc
 
 [handlers]
-pre-install=/usr/bin/rauc_downgrade_barrier.sh
+pre-install=/usr/lib/rauc/rauc-pre-install.sh
+post-install=/usr/lib/rauc/rauc-post-install.sh
 
 [keyring]
-path=ca.cert.pem
+path=@RAUC_KEYRING_FILE@
 
 # Bootloader
 [slot.bootloader.0]
-device=/dev/mmcblk2
+device=/dev/mmcblk@EMMC_DEV@
 type=boot-emmc
 
 # System A
 [slot.rootfs.0]
-device=/dev/mmcblk2p5
+device=/dev/mmcblk@EMMC_DEV@p5
```

Comment:
meta-ampliphy use the variable ROOTFS_0_DEV for the partition, because file encryption change the partition to the device mapper /dev/dm-0.
Why do you use the EMMC_DEV here?